Readme.md

```diff
@@ -60,7 +60,8 @@ Here's an overview of all of the scripts this project makes available, categoriz
 
 1. [Styles](Lombiq.NodeJs.Extensions/Docs/Styles.md)
 2. [JavaScript](Lombiq.NodeJs.Extensions/Docs/JavaScript.md)
-3. [Markdown](Lombiq.NodeJs.Extensions/Docs/Markdown.md)
+3. [Asset Copying](Lombiq.NodeJs.Extensions/Docs/AssetCopying.md)
+4. [Markdown](Lombiq.NodeJs.Extensions/Docs/Markdown.md)
```

Comment:
Why did you insert Asset Copying in the middle of the list? Does this order represent something?
I think this list doesn't need to be numbered, and it could be just sorted alphabetically. What is your opinion?

Reply:
I was thinking that the first three process assets in a project for publishing, while 4. does static analysis. So I wanted to group them according to that.

Reply:
That seems arbitrary. You could make it explicit with a sublist or something.

Also why are publishing items in the Styles, JavaScript, Asset Copying order?

Reply:
I've come to like your suggestion of using an unordered, alphabetically sorted list.

Reply:
What a beautiful staircase:

```markdown
- [Asset Copying](Lombiq.NodeJs.Extensions/Docs/AssetCopying.md)
- [JavaScript](Lombiq.NodeJs.Extensions/Docs/JavaScript.md)
- [Markdown](Lombiq.NodeJs.Extensions/Docs/Markdown.md)
- [Styles](Lombiq.NodeJs.Extensions/Docs/Styles.md)
```

😁

Reply:
Ah, so that's what they meant by "waterfall model"! 🙃

Reply:
You said you liked my suggestion, but then didn't commit it?

Reply:
Indeed, for some reason it ended up in a stash 😏